src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import './App.css';
+import MathApp from './MathApp';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <MathApp />
```

Comment:
- [ ] Kindly revert the change and render the `App` component inside `index.js` instead of `MathApp`, and inside the `App` component render the `Calculator` component as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Once%20the%20Calculator%20is%20ready%2C%20import%20it%20and%20use%20it%20in%20your%20main%20component%20(App.js).).